de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/snippet/dialog/EnumerationCreationDialog.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.requirements.ui.snippet.dialog;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.edit.domain.EditingDomain;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.ui.forms.widgets.FormToolkit;
+
+import de.dlr.sc.virsat.model.extension.requirements.ui.snippet.UiSnippetTableEnumerationDefinitionLiteralsEnumerationLiteral;
+
+/**
+ * @author fran_tb
+ *
+ */
+public class EnumerationCreationDialog extends AUiSnippetCreationDialog {
+	
+	protected FormToolkit toolkit;
+	protected EditingDomain editingDomain;
+	protected EObject initModel;

```

Comment:
Do we still need these variables here?
Checking AUiSnippetCreationDialog it seems they are already stored there.